docs/index.html

```diff
@@ -535,20 +535,20 @@ <h2 class="heading settled" data-level="2" id="terminology"><span class="secno">
     <dt data-md><dfn class="dfn-paneled" data-dfn-type="dfn" data-noexport id="presentation">presentation</dfn>
     <dd data-md>
      <p><a href="https://www.w3.org/TR/vc-data-model/#presentations-0">Presentation</a> is a mechanism used to share one or more credentials with a given party.</p>
    </dl>
    <h2 class="heading settled" data-level="3" id="formatting"><span class="secno">3. </span><span class="content">Formatting</span><a class="self-link" href="#formatting"></a></h2>
-   <p>The <a data-link-type="biblio" href="#biblio-vc-data-model">[VC-DATA-MODEL]</a> utilizies the <a href="https://www.w3.org/TR/ld-glossary/#linked-data">JSON Linked Data interchange format</a>. The specification allows for other formats, such as standard JSON with JSON Schema but provides limited examples. In the <a href="https://www.w3.org/TR/vc-data-model/#data-schemas">Data Schemas section</a>, <i>JSON-SCHEMA-2018</i> validation is noted explicitly. This specification does not <b>require</b> the use of <a data-link-type="biblio" href="#biblio-json-ld">[JSON-LD]</a>, though some may prefer to utilize <a data-link-type="dfn" href="#data-integrity" id="ref-for-data-integrity">Data Integrity</a> Proofs which this specification supports. If it becomes evident that it would be useful to include <a data-link-type="biblio" href="#biblio-json-ld">[JSON-LD]</a> or another format that decision would be made in a revisal draft at a later date.</p>
-   <p>The Verifiable Credentials data model relies heavily upon standard JSON with validation provided by <a data-link-type="biblio" href="#biblio-json-schema">[JSON-SCHEMA]</a>. The data model embeds <a data-link-type="biblio" href="#biblio-json-schema">[JSON-SCHEMA]</a> documents inside a larger document that contains useful metadata about a given credential schema.</p>
+   <p>The <a data-link-type="biblio" href="#biblio-vc-data-model" title="Verifiable Credentials Data Model 1.0">[VC-DATA-MODEL]</a> utilizies the <a href="https://www.w3.org/TR/ld-glossary/#linked-data">JSON Linked Data interchange format</a>. The specification allows for other formats, such as standard JSON with JSON Schema but provides limited examples. In the <a href="https://www.w3.org/TR/vc-data-model/#data-schemas">Data Schemas section</a>, <i>JSON-SCHEMA-2018</i> validation is noted explicitly. This specification does not <b>require</b> the use of <a data-link-type="biblio" href="#biblio-json-ld" title="JSON-LD 1.1: A JSON-based Serialization for Linked Data">[JSON-LD]</a>, though some may prefer to utilize <a data-link-type="dfn" href="#data-integrity" id="ref-for-data-integrity">Data Integrity</a> Proofs which this specification supports. If it becomes evident that it would be useful to include <a data-link-type="biblio" href="#biblio-json-ld" title="JSON-LD 1.1: A JSON-based Serialization for Linked Data">[JSON-LD]</a> or another format that decision would be made in a revisal draft at a later date.</p>
```

Comment:
```suggestion
   <p>The <a data-link-type="biblio" href="#biblio-vc-data-model" title="Verifiable Credentials Data Model 1.0">[VC-DATA-MODEL]</a> utilizies the <a href="https://www.w3.org/TR/ld-glossary/#linked-data">JSON Linked Data interchange format</a>. The specification allows for other formats, such as standard JSON with JSON Schema, but provides limited examples. In the <a href="https://www.w3.org/TR/vc-data-model/#data-schemas">Data Schemas section</a>, <i>JSON-SCHEMA-2018</i> validation is noted explicitly. This specification does not <b>require</b> the use of <a data-link-type="biblio" href="#biblio-json-ld" title="JSON-LD 1.1: A JSON-based Serialization for Linked Data">[JSON-LD]</a>, though some may prefer to utilize <a data-link-type="dfn" href="#data-integrity" id="ref-for-data-integrity">Data Integrity</a> Proofs which this specification supports. If it becomes evident that it would be useful to include <a data-link-type="biblio" href="#biblio-json-ld" title="JSON-LD 1.1: A JSON-based Serialization for Linked Data">[JSON-LD]</a> or another format that decision would be made in a revised draft at a later date.</p>
```